components/match2/commons/match_group_input.lua

```diff
@@ -250,4 +251,29 @@ function MatchGroupInput.fetchStandaloneMatch(matchId)
 	end)
 end
 
+--[[
+Coerces the opponent specification part of an opponent record. If the opponent
+spec is not present, then the record will be treated as an input args table,
+from which the spec will be parsed using Opponent.readOpponentArgs.
+]]
+function MatchGroupInput.coerceOpponentRecord(record)
```

Comment:
Coerce? Why not use `force`?

Reply:
i've heard type coercion used a lot more than type forcing

Reply:
It's a term in JS, but in this case you're not using `coerce` along with type, so I prefer `force`

Reply:
I replaced it with `MatchGroupInput.mergeRecordWithOpponent` which uses a different approach.